testproject/Assets/Tests/Runtime/Support/SpawnRpcDespawn.cs

```diff
@@ -66,7 +83,7 @@ private void RunTest()
             Debug.Log("Running test...");
             GetComponent<NetworkObject>().Spawn();
             IncrementUpdateCount();
-            GetComponent<NetworkObject>().Despawn(false);
+            Destroy(gameObject);
```

Comment:
I'm curious about this change. Is this needed for this fix or an unrelated behaviour change. It's quite likely to be a better idea to destroy, but I wonder if it is part of the fix.